pom.xml

```diff
@@ -94,7 +94,7 @@
           </execution>
         </executions>
         <configuration>
-          <createDependencyReducedPom>true</createDependencyReducedPom>
+          <createDependencyReducedPom>false</createDependencyReducedPom>
```

Comment:
this means the pom we generate will have the shaded/relocated jars as deps, that's probably not what you want as the jcraft class will be on the classpath twice as a dep and in the uberjar.
Probably best to add the dependencyReduce pom to gitignore.

Might also want to add `<promoteTransitiveDependencies>true</promoteTransitiveDependencies>` the generated pom will be a drop in replacement for your uberjar.

Reply:
I couldn't find how this POM is used, if it doesn't get uploaded to central then there is no point generating it. I was browsing for examples but didn't find anything relevant. I'll take a look again, if you know how we can do this please advise

Reply:
ok, I've put this back

Reply:
on the usage if you follow the output of `mvn install` you'll see it copies the reduduced pom to your local maven repo, replacing the normal pom.